### DISCUSSION TITLE
Using TVTap/TVMob or Swiftstreamz?

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#119] [Show and tell]
Any possibilities of using the links from those free apps as sources? Only problem is that they're tokenized.

**Comment by SSyar** [comment]
There were used to playlists on http://stream.nmbs.top/ for various categories/countries from swiftstreamz but now they don't work

**Reply by thedarkdragon11** [reply]
Yeah... I wonder if it can be replicated for Swiftstreamz and TVTap/TVMob?

**Reply by SSyar** [reply]
Swfitstreams is a fantastic app its just the interface which is not likeable to use for TV Viewing .. as switching of channel is done by going back to channel selection and then selecting the desired channel

**Reply by BellezaEmporium** [reply]
The only possible issue I could see with these kind of apps is that I think they often rely on XTream-Codes to deliver their streams through the app, which is not something I would recommend using.

**Reply by thedarkdragon11** [reply]
But is it possible? About the sources, I can't speak for all their channels (especially TVMob) but I'm certain that most of them are from direct sources, like the Philippine channels from Swiftstreamz for example.

**Reply by BellezaEmporium** [reply]
I can check the app if I either decompile the APK or if I use Frida to sniff the network, but I cannot guarantee any result from it.

**Reply by thedarkdragon11** [reply]
Hopefully they'll work.

**Reply by BellezaEmporium** [reply]
Try changing your links from http://stream.nbms.top to http://alfabetagame.ddns.net/.

Geo News PK is located on http://alfabetagama.ddns.net/pak2020/geonewspk/playlist.m3u8?wmsAuthSign= *insert*

POST token is being asked on a HTTP link : http://62.210.172.84/tokenv2.2/token01.php -> wmsAuthSign

None of them seems to work, though.

**Comment by thedarkdragon11** [comment]
I wonder if this is still possible? http://stream.nmbs.top/ was a thing before.
